libandroid-navigation/src/main/java/org/maplibre/navigation/android/navigation/v5/navigation/NavigationService.kt

```diff
@@ -41,7 +41,7 @@ open class NavigationService : Service() {
      * Only should be called once since we want the service to continue running until the navigation
      * session ends.
      */
-    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
+    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
```

Comment:
Correct! Can be `NULL`. 

> The Intent supplied to Context. startService, as given. This may be null if the service is being restarted after its process has gone away, and it had previously returned anything except START_STICKY_COMPATIBILITY.